.circleci/run-tests.sh

```diff
@@ -100,6 +100,9 @@ case $1 in
     chipyard-sodor)
         run_asm ${mapping[$1]}
         ;;
+    chipyard-ibex)
+        run_bmark ${mapping[$1]}
```

Comment:
Wait... why does this pass? Shouldn't the bmarks be RV64... not RV32 (and thus should break)?